modules/checkbox/react/spec/Checkbox.spec.tsx

```diff
@@ -1,102 +1,134 @@
 import * as React from 'react';
-import {mount} from 'enzyme';
+import {render, fireEvent} from '@testing-library/react';
 import Checkbox from '../lib/Checkbox';
 import ReactDOMServer from 'react-dom/server';
 import {axe} from 'jest-axe';
 import FormField from '@workday/canvas-kit-react-form-field';
 
+const label = 'Test Checkbox';
+
 describe('Checkbox', () => {
   const cb = jest.fn();
   afterEach(() => {
     cb.mockReset();
   });
 
-  test('render an checkbox input with id', () => {
-    const component = mount(<Checkbox id="myCheckbox" onChange={cb} />);
-    expect(component.find('input').props().id).toBe('myCheckbox');
-    component.unmount();
-  });
+  describe('when rendered', () => {
+    describe('with an id', () => {
+      test('should render a checkbox input with id', () => {
```

Comment:
I restructured these `describe`/`test` functions to be behavior based and read more like specs (similar to how we do our Cypress tests). Hopefully everyone likes this!

Reply:
👍 My only nit is using `it` instead of `test` for specifications